tests/test_vega.py

```diff
@@ -267,3 +264,433 @@ def test_fill_anchor_in_string(tmp_dir):
+                {
+                    "rev": "B",
+                    "acc": "0.1",
+                    "filename": "test",
+                    "field": "acc",
+                    "step": 2,
+                },
+            ],
+            "acc",
+            [{"filename": "test", "field": "acc"}],
+            ["rev", "acc", "step"],
+            {},
+        ),
+        (
+            [
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.05",
+                    "filename": "test",
+                    "field": "acc",
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.04",
+                    "filename": "train",
+                    "field": "acc_norm",
+                    "step": 1,
+                },
+            ],
+            "dvc_inferred_y_value",
+            [
+                {"filename": "test", "field": "acc"},
+                {"filename": "test", "field": "acc_norm"},
+            ],
+            ["rev", "dvc_inferred_y_value", "step", "field"],
+            {
+                "field": "field",
+                "scale": {"domain": ["acc", "acc_norm"], "range": [[1, 0], [8, 8]]},
+                "legend": {
+                    "symbolFillColor": "transparent",
+                    "symbolStrokeColor": "grey",
+                },
+            },
+        ),
+        (
+            [
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.05",
+                    "filename": "test",
+                    "field": "acc",
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.04",
+                    "filename": "train",
+                    "field": "acc",
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.02",
+                    "filename": "test",
+                    "field": "acc_norm",
+                    "step": 1,
+                },
+            ],
+            "dvc_inferred_y_value",
+            [
+                {"filename": "test", "field": "acc_norm"},
+                {"filename": "test", "field": "acc"},
+                {"filename": "train", "field": "acc"},
+            ],
+            ["rev", "dvc_inferred_y_value", "step", "filename::field"],
+            {
+                "field": "filename::field",
+                "scale": {
+                    "domain": ["test::acc", "test::acc_norm", "train::acc"],
+                    "range": [[1, 0], [8, 8], [8, 4]],
+                },
+                "legend": {
+                    "symbolFillColor": "transparent",
+                    "symbolStrokeColor": "grey",
+                },
+            },
+        ),
+    ),
+)
+def test_partial_filled_template(
```

Comment:
[I] It is going to be necessary to extend these tests to cover all of the new templates.